### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by hjacque**
Naming, spacing and good practices.

### CODE REVIEW

File: components/baseMap.js [hunk 1]
```diff
@@ -55,26 +55,18 @@ const BaseMap = ({ mapLocations }) => {
           position={location.htmlcoordinates}
           icon={locationIcon}
           eventHandlers={{
-            click: (e) => {
-              handleMarkerClick(location)
-            },
+            click: (_e) => handleMarkerClick(location),
```
Comment (hjacque) [inline components/baseMap.js]: recommandation for indicating a function argument is unused in JavaScript

File: components/mapLegend.js [hunk 2]
```diff
@@ -1,51 +1,45 @@
...
+import React, { useState } from "react";
+import AnimateHeight from "react-animate-height";
+import { animated, useSpring } from "react-spring";
+import styles from "../styles/legend.module.css";
 
 export default function MapLegend({ location }) {
-    const [height, setHeight] = useState(0);
-    const [open, setOpen] = useState(false);
+  const [height, setHeight] = useState(0);
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (hjacque) [inline components/mapLegend.js:10]: naming recommandation for indicating a variable is boolean in JavaScript

File: components/mapLegend.js [hunk 3]
```diff
@@ -1,51 +1,45 @@
...
 
-    const clickHandler = (e) => {
-        setOpen(!open)
-        setHeight(height === 0 ? 'auto' : 0)
-    }
+  const clickHandler = (e) => {
+    setIsOpen(!isOpen);
+    setHeight(height === 0 ? "auto" : 0);
```
Comment (hjacque) [inline components/mapLegend.js:14]: height type weird to me: is it a string or a number? maybe we should switch to typescript now xD

Comment (HDDLabs) [reply components/mapLegend.js:14]: Yes, switching to typescript is on the roadmap.

File: .env.example [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# PLEASE COPY THIS FILE TO A NEW .env FILE
+# THEN EDIT AND SET THE VARIABLES
+
+NEXT_PUBLIC_WIKI_API_KEY=your_next_public_wiki_api_key
```
Comment (HDDLabs) [inline .env.example:4]: What is the purpose of this file?

File: lib/wiki.js [hunk 5]
```diff
@@ -52,7 +52,7 @@ export async function getWikiLocationsData() {
     query: locationsQuery,
   });
 
-  const updatedArray = data.pages.list.map((item) => {
+  const updatedArray = data?.pages?.list.map((item) => {
```
Comment (HDDLabs) [inline lib/wiki.js:55]: Nice, I didn't realize JS had optionality/optional chaining 🙌

**Review by HDDLabs — APPROVED** [review]

File: lib/wiki.js (hunk 5 above)
Comment (hjacque) [reply lib/wiki.js:55]: LFG

File: .env.example (hunk 4 above)
Comment (hjacque) [reply .env.example:4]: so user can setup their own .env file

**Pull request merged**